doc/source/whatsnew/v0.22.0.txt

```diff
@@ -44,7 +44,7 @@ Other API Changes
 - All-NaN levels in ``MultiIndex`` are now assigned float rather than object dtype, coherently with flat indexes (:issue:`17929`).
 - :class:`Timestamp` will no longer silently ignore unused or invalid `tz` or `tzinfo` arguments (:issue:`17690`)
 - :class:`CacheableOffset` and :class:`WeekDay` are no longer available in the `tseries.offsets` module (:issue:`17830`)
--
+- all methods which use `_get_handle()` internally, now support "infer" as compression argument (eg. `to_csv()`)
```

Comment:
Can you rewrite this a bit from a user standpoint of view? (who does not know about `_get_handle`). So if it is only `to_csv`, just mention that `to_csv` gained the option of 'infer' for the `compression` keyword.

(you can also put it in the 'Other enhancements' section instead of api changes.